pkg/pxc/queries/queries.go

```diff
@@ -317,6 +313,11 @@ func (p *Database) PresentInHostgroups(host string) (bool, error) {
 
 func (p *Database) PrimaryHost() (string, error) {
 	var host string
+
+	// value of writer group is hardcoded in ProxySQL config inside docker image
+	// https://github.com/percona/percona-docker/blob/pxc-operator-1.3.0/proxysql/dockerdir/etc/proxysql-admin.cnf#L23
+	var writerID = 11
+
```

Comment:
why do we use var not const?

Reply:
Thanks for catching! Returned to the original type and place since there is no need to be variable. 7d4e5cebaa4e1e90531adbf1b48ac193375f7632